pkg/internal/checkup/executor/trex-console.go

```diff
@@ -0,0 +1,69 @@
+package executor
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+type trexConsole struct {
+	podClient     podExecuteClient
+	namespace     string
+	name          string
+	containerName string
+}
+
+func NewTrexConsole(client podExecuteClient, namespace, name, containerName string) *trexConsole {
```

Comment:
As we've discussed offline, the methods of this object seems to work without a pointer receiver, because they don't change the object's state.
Please consider using an object receiver, and returning an object.

Reply:
DONE